src/PageContent/AssignVolunteers/ConfirmationCard.js

```diff
@@ -35,41 +36,45 @@ class ConfirmationCard extends Component {
                     </div>
                     <div className="delivery-details">
                         <h5>Student Deliverers</h5>
-                        <h4>{this.state.studentGroup}</h4>
-                        <h6 className="det">{this.props.deliverer1.name} ({this.props.deliverer1.phone})</h6>
-                        <h6 className="det">{this.props.deliverer2.name} ({this.props.deliverer2.phone})</h6>
+                        <h4>{delivererGroup.group}</h4>
```

Comment:
I don't think there is actually a `delivererGroup` in the `delivery` object you filled

Reply:
I realize now this was kind of dumb of me... The delivererGroup is the currently-logged-in deliverer. I don't need to tell them who they are...